api/nakadi-event-bus-api.yaml

```diff
@@ -2328,6 +2386,62 @@ definitions:
         description: |
           Number of milliseconds that Nakadi stores events published to this event type.
 
+  EventTypeAuthorization:
+    type: object
+    description: |
+      Authorization section for an event type. This section defines three access control lists: one for producing
+      events ('write'), one for consuming events ('read'), and one for administering an event type ('admin').
+      Regardless of the values of the authorization properties, administrator accounts will always be authorized.
+    properties:
+      admin:
+        type: array
```

Comment:
@lmontrieux wrote (in a top-level comment):

> As far as I can tell, Zally only reports one new issue caused by the introduction of authorization:
SHOULD – Array names should be pluralized
#/definitions/EventTypeAuthorization: 'admin','read','write'

> I believe that the names 'admin', 'read', and 'write' should not be changed in this case, as they are widely used in authorization.

I guess the point is that the value of those properties are not an "admin", a "read" or a "write", but lists of authorizations. So maybe better names would be `admin_authorizations`, `read_authorizations` and `write_authorizations`?